### DISCUSSION TITLE
How do I pass properties to a `ce-app`?

### DISCUSSION THREAD

**Discussion by Shinigami92** [baiwusanyu-c/unplugin-vue-ce#124] [Q&A]
Can we get an example of a Vue 3 + Vite 5 + ce-app where it is demonstrated how to pass properties from outside into the custom element component?

**Comment by mrcego** [comment]
> Can we get an example of a Vue 3 + Vite 5 + ce-app where it is demonstrated how to pass properties from outside into the custom element component?

Like a [root component](https://github.com/unplugin/unplugin-vue-ce/tree/master/play/ce-app)?
